pipe_viewer C++ assertion during OnHoverRedraw on MacOS

After fixing #113 Argos/PipeViewer will launch and run on MacOS but any time you move the mouse across the canvas, you will get a gagillion messages that repeatedly say:
```
Layout loaded with 112 elements in 0.15770912170410156s
Traceback (most recent call last):
  File "/Volumes/cs/map/helios/pipeViewer/pipe_view/gui/layout_canvas.py", line 394, in OnHoverRedraw
    dc = wx.BufferedPaintDC(self, self.__buffer)
wx._core.wxAssertionError: C++ assertion "window->MacGetCGContextRef() != NULL || IsGLCanvas(window)" failed at /Users/user917190/miniconda3/conda-bld/wxpython_1587407908580/work/ext/wxWidgets/src/osx/carbon/dcclient.cpp(205) in wxPaintDCImpl(): using wxPaintDC without being in a native paint event
```

@TopChangwatchaiSiFive Looks like the `OnHoverRedraw` event handler needs to be rewritten to be cross-platform.  Let me know if you want/need help with this.  I need to go back to looking at FireSim

--- a/helios/pipeViewer/pipe_view/gui/layout_canvas.py
+++ b/helios/pipeViewer/pipe_view/gui/layout_canvas.py
@@ -391,7 +391,7 @@ class Layout_Canvas(wx.ScrolledWindow):
         if self.__hover_preview.show:
             dc.SetFont(self.__fnt_layout)
             self.DoHoverTextDraw(dc)
-        dc = wx.BufferedPaintDC(self, self.__buffer)
+        dc = wx.BufferedDC(wx.ClientDC(self), self.__buffer)
 
     # # Returns Hover Preview
     def GetHoverPreview(self):

